lib/renderer/security-warnings.ts

```diff
@@ -101,7 +101,7 @@ const warnAboutInsecureResources = function () {
 
   const warning = `This renderer process loads resources using insecure
   protocols.This exposes users of this app to unnecessary security risks.
-  Consider loading the following resources over HTTPS or FTPS. \n ${resources}
+  Consider loading the following resources over HTTPS or FTPS. \n${resources}
   \n ${moreInformation}`
```

Comment:
Should the extra space be removed here too?

Reply:
Yes it should, I’ll do it later tonight.

Although visually it won’t make a difference since moreInformation starts off with a newline :)